#include "catch2/catch.hpp"

TEST_CASE("Test fault process", "[process]")
{
    //do some test

}
